### PR TITLE
UI-Guidelines: Added Colors and applied coding standards to CSS

### PR DESCRIPTION
**Pull request by inakijv**
Most important aspects:

- Added Colors to Resources section.
- Five groups included: Primary, Secondary, Text, Background and Notification colors.
- Listed colors within each group might be modified and/or updated, based on design or development requirements.

screen shot 2017-02-09 at 4 17 09 pm

### CODE REVIEW

File: ui-guidelines/css/resources-colors.css [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    background-color: #006094;
+}
+.c-neutral {
+    background-color: #9b9b9b;
+}
+.c-neutral-hover {
+    background-color: #667076;
+}
+.c-negative {
+    background-color: #E85365;
```
Comment (SylvainGuittard) [inline ui-guidelines/css/resources-colors.css]: CS: hex code should be lowercase

Comment (inakijv) [inline ui-guidelines/css/resources-colors.css]: Thanks! :-) Working on it now

File: ui-guidelines/css/guidelines.css [hunk 2]
```diff
@@ -143,28 +136,10 @@ p {
...
-}
-iframe {
-	border-radius: 4px;
-	border: 1px solid #fce3da;
 }
-pre {
-	font-size: 1em;
-	font-weight: bold;
+hr {
+	margin-top: 0px;
```
Comment (SylvainGuittard) [inline ui-guidelines/css/guidelines.css]: CS: `margin-top: 0;`

File: ui-guidelines/css/resources-colors.css [hunk 3]
```diff
@@ -0,0 +1,167 @@
+.color-row {
+	display: flex;
```
Comment (SylvainGuittard) [inline ui-guidelines/css/resources-colors.css]: CS: remove the indent

File: ui-guidelines/css/resources-colors.css [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+    justify-content: flex-start;
+    flex-wrap: wrap;
+    align-content: space-around;
+    margin-bottom: 2rem;
+}
+.color-group {
+	margin: .75rem 1.5rem 0 0;
```
Comment (SylvainGuittard) [inline ui-guidelines/css/resources-colors.css]: CS: indent

File: ui-guidelines/css/resources-colors.css [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+}
+.color-box {
+	height: 8rem;
```
Comment (SylvainGuittard) [inline ui-guidelines/css/resources-colors.css]: CS: indent (and below)

File: ui-guidelines/css/resources-colors.css [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+	width: 11rem;
+    margin-bottom: 1rem;
+	border: 1px solid transparent;
+	border-radius: 4px;
+}
+.color-box-lightColor {
+    border: 1px solid #dddddd;
```
Comment (SylvainGuittard) [inline ui-guidelines/css/resources-colors.css]: You can change to ` border: 1px solid #ddd;`

File: ui-guidelines/css/resources-colors.css [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+}
+.color-label {
+    margin-top: -.75rem;
+	font-size: 1.15rem;
```
Comment (SylvainGuittard) [inline ui-guidelines/css/resources-colors.css]: CS: indent and below

File: ui-guidelines/css/resources-colors.css [hunk 8]
```diff
@@ -0,0 +1,167 @@
...
+	font-family: 'Roboto', sans-serif;
+	font-weight: 400;
+}
+.label-code {
+	display: inline-block;
```
Comment (SylvainGuittard) [inline ui-guidelines/css/resources-colors.css]: CS: indent

**Pull request merged**